CONTRIBUTING.md

```diff
@@ -0,0 +1,24 @@
+# Decentralized Identifier Working Group
```

Comment:
Tsk tsk. Copying and pasting. The horrors... ;)

Reply:
Oops:-)

Done